engine/src/contract_methods/connector/internal.rs

```diff
@@ -444,6 +444,11 @@ pub fn ft_metadata<I: IO + Copy>(mut io: I) -> Result<(), ContractError> {
     Ok(())
 }
 
+pub fn get_bridge_prover<I: IO + Copy>(io: I) -> Result<(), ContractError> {
+    EthConnectorContract::init(io)?.get_bridge_prover();
```

Comment:
In the original code this value was returned via `io.return_output`. Perhaps a test of this function should be added as well?

Reply:
Good catch 👍🏻